go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/cockroachdb/swiss
+
+go 1.22.0
```

Comment:
The go directive sets the minimum version of Go required to use this module. Does it need to be this high? `runtime_go1.20.go` would imply that this can drop down to go1.20, which should also have all of the support for generics that you need.

Reply:
Ah, I misremembered what this directive does. I've set it to `go 1.20`.